docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -182,6 +182,24 @@ git commit -am "Next is $NEXT_SNAPSHOT"
 # Send a pull request
 ----
 
+[[release-announcement]]
+== Write the release announcement blog post
+
+Taking inspiration from previous release announcement blog posts, we need to write one for the release at hand.
+
+The time before closing the vote is perhaps the most suitable for writing the release announcement.
+In that way, the announcement link can be shared with Quarkus team via
+xref:contributor-guide/release-guide.adoc#quarkus-platform[Platform pull request]
+and Quarkus team will mention the new Camel Quarkus release in their announcement thus reaching the whole Quarkus audience.
+
+Please, keep in mind a few things when preparing the pull request for the camel-website:
```

Comment:
> preparing the pull request

Are we intending for the PR to actually be opened at this stage? If it's done before the vote is closed, then it'd need to be in draft status before all of the bits have been released.

Reply:
TBH, I have been interpreting "preparing" as "local preparing", but yes, if somebody opens the PR before the vote is closed, it should be a draft and it can only be merged after the vote.

Should we be more explicit about that here, or we rather move the whole announcement paragraph right before the Platform upgrade?

Reply:
> move the whole announcement paragraph right before the Platform upgrade?

Yes, would probably be safer that way.

Reply:
Like the website pr remains in draft, when the vote is ok, the website pr is merged. Then we wait for the CQ blog post to be published, and  we include the CQ blog post url in the platform pr. Could it block platform release if we have issues with blog post ?

Taking a look at typical information included in the upcoming Camel Quarkus 3.14.0 blog post:
 * Camel version 4.7.0 [we know]
 * Quarkus version 3.14.0 [we know]
 * Release note info https://camel.apache.org/releases/q-3.14.0/ [we know]
 * Camel release announcement https://camel.apache.org/blog/2024/07/RELEASE-4.7.0/ [could be challenging some monthes]
 * Quarkus release announcement https://quarkus.io/blog/quarkus-3-13-0-released/  [could be predicted]

On the top of my head, I would vote for keeping less coupling. The ideal situation would be to have a kind of conditional link in the quarkus announcement. #if[cq announcement is ready], then link to it ;)